content/applications/sales/sales/send_quotations/pdf_quote_builder.rst

```diff
@@ -0,0 +1,231 @@
+   :guilabel:`partner_id_name`).
+
+   .. image:: pdf_quote_builder/pdf-quote-builder-sample.png
+      :align: center
+      :alt: PDF quote being built using common dynamic placeholders.
+
+Once the PDF file(s) are complete, save them to the computer's hard drive, and proceed to upload
+them to Odoo, via :menuselection:`Sales app --> Configuration --> Settings --> PDF Quote builder`.
+
+Then, proceed to upload the created PDF in the :guilabel:`Header pages` or :guilabel:`Footer pages`
+field.
+
+Once the upload(s) are complete, click :guilabel:`Save`.
+
+Add PDF to product
+==================
+
+In Odoo *Sales*, it's also possible to add a custom PDF to a product. When a PDF is added to a
+product, and that product is used in a quotation, that PDF is also inserted in the final PDF.
+
+To add a custom PDF to a product, start by navigating to :menuselection:`Sales app --> Products -->
+Products`, and selected the desired product to which a custom PDF should be added.
+
+.. note::
+   A document could also be added to a product variant, instead of a product. Be aware that, if
+   there are documents on a product *and* on its variant, **only** the documents in the variant are
+   shown.
+
+On the product page, click the :guilabel:`Documents` smart button at the top of the page.
+
+.. image:: pdf_quote_builder/documents-smart-button.png
+   :align: center
+   :alt: The Documents smart button on a product form in Odoo Sales.
+
+Doing so reveals a separate :guilabel:`Documents` page for that product, wherein files related to
+that product can be uploaded. From this page, either click :guilabel:`New` or :guilabel:`Upload`.
+
+Clicking :guilabel:`Upload` instantly provides the opportunity to upload the desired document.
+
+.. note::
+    If :guilabel:`Upload` is clicked from the :guilabel:`Documents` page, Odoo uploads the desired
+    file, and reveals a documents form, with that file already loaded in the :guilabel:`File
+    Content` field.
+
+    In other words, clicking :guilabel:`Upload` directly from the :guilabel:`Documents` page results
+    in a documents form that can be further configured and customized.
+
+Clicking :guilabel:`New` reveals a blank documents form, in which the desired PDF can be uploaded,
+as well, via the :guilabel:`Upload your file` button on the form, located in the :guilabel:`File
+Content` field.
+
+.. image:: pdf_quote_builder/blank-document-form.png
+   :align: center
+   :alt: A standard document form with various fields for a specific product in Odoo Sales.
+
+Various information and configurations related to the uploaded document can be modified here.
+
+The first field on the documents form is for the :guilabel:`Name` of the document, and it's
+greyed-out. If a PDF has already been uploaded, or will be uploaded via the :guilabel:`Upload your
+file` button on the form, the :guilabel:`Name` field is auto-populated with the name of the PDF.
```

Comment:
```suggestion
greyed-out until a document is uploaded. Once a PDF has been uploaded, the :guilabel:`Name` field is auto-populated with the name of the PDF, and it can then be edited.
```